doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -289,14 +281,20 @@ Architecture they must be implemented as per the following specifications:
 ## 4.9 Kubernetes workloads
 
 In order for the Kubernetes workloads to be conformant with the Reference
-Architecture they must be implemented as per the following specifications:
+Architecture they must implement the parameter values as per the following specifications:
 
-|Ref|Specification|Details|Requirement Trace|
+|Ref|Parameter Group|Details|Requirement Trace|
```

Comment:
Mention the parameter groups in the specs themselves, as there will be other specifications not related to parameter groups to come in the next release.
```suggestion
|Ref|Specification|Details|Requirement Trace|
```